src/commands/kv/namespace/create.rs

```diff
@@ -7,9 +7,15 @@ use crate::settings::global_user::GlobalUser;
 use crate::settings::target::Target;
 use crate::terminal::message;
 
-pub fn create(target: &Target, user: GlobalUser, title: &str) -> Result<(), failure::Error> {
+pub fn create(
+    target: &Target,
+    env: Option<&str>,
+    user: GlobalUser,
+    binding: &str,
+) -> Result<(), failure::Error> {
     let client = kv::api_client(user)?;
 
+    let title = format!("{}-{}", target.name, binding);
```

Comment:
this will result in a no-env title that ends with `-` like `STATIC_ASSETS-`

Reply:
This logic actually should be fine; doesn't #551 have you use the binding and target name instead of env? Both values going into the formatting string are guaranteed to exist.

Reply:
yes. 🤦‍♀